File: bayesmendel-comparator/r/R/model_registry.R
# Declares the three BayesMendel models this demo compares.
#
# Keeping the registry in one place means adding a fourth model is a
# one-line edit -- no presenter, no builder, no client needs to change.

#' The BayesMendel models compared by this demo, in display order.
#' @return Character vector of model names.
bayesmendel_models <- function() {
  c("BRCAPRO", "MMRpro", "PancPRO")
}

File: bayesmendel-comparator/python/src/bayesmendel_comparator/model_registry.py
"""Declares the three BayesMendel models this demo compares.

Keeping the registry in one place means adding a fourth model is a
one-line edit — no presenter, no builder, no client needs to change.
"""

from __future__ import annotations

BAYESMENDEL_MODELS: tuple[str, ...] = ("BRCAPRO", "MMRpro", "PancPRO")
